### ISSUE TITLE
Installing Google Plattform Tools via winget seems to not work as expected

### ISSUE THREAD

**Issue by PostboxRetinal** [microsoft/winget-pkgs#103349]
### Brief description of your issue

Installed Plattform tools (adb and fastboot) via winget as normal (winget install --id Google.PlatformTools). Restarted terminal.
Whenever I need to run these tools I try adb, and terminal isn't returning something at all, but, when I type winget adb it seems to work, but this isn't the normal way of work

![yt03QXu5uY](https://user-images.githubusercontent.com/78338192/234039196-5a293736-1b0d-4758-b0ab-a5f0ebb73b67.png)

### Steps to reproduce

1. Open a Windows Terminal as admin
2. Install adb via winget (winget install --id Google.PlatformTools)
3. Restart Terminal
4. Try to run adb, it wont show anything unless you call winget first

### Expected behavior

Type adb and use the application as a normal installation adding manually the tool to path

### Actual behavior

Just runs when you write it as an argument after winget
i.e `winget adb` and prints `Unrecognized command: 'adb'` but eventually seems to work

![J1M0NMpy8m](https://user-images.githubusercontent.com/78338192/234038498-b86d539c-c560-4d3d-9b4b-3d5389b14bcf.png)

### Environment

```shell
Windows Package Manager v1.4.10173
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.22621.1555
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.19.10173.0

Logs: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir

User Settings: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\settings.json

Links
---------------------------------------------------------------------------
Privacy Statement   https://aka.ms/winget-privacy
License Agreement   https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage            https://aka.ms/winget
Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale
```

**Comment by Minh-AnhHuynh** [comment]
I have the same issues and currently it's still not resolved.

**Comment by pheizaut** [comment]
Any of you maybe likely be using PowerShell, run this to reload EnvVars in PW. Restarting terminal wasn't enough but this was.
`$env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User") 
`

**Comment by Frioo** [comment]
Still not resolved! Facing the same bug.

**Comment by zyc63959** [comment]
hi. i get a way to resolve it. firstly i installed  scrcpy not this. so i create a bat that's name is "adb.bat" and a ps file that's name is "adb.ps1"  in "winget/links" folder. bat'code is `@echo off
call powershell.exe -NoProfile -ExecutionPolicy Bypass -File "C:\Users\yingc\AppData\Local\Microsoft\WinGet\Links\adb.ps1" %*`
ps'code is `
param(
    [string]$Command,
    $Params
)
$filePath = (Get-Item "C:\Users\yingc\AppData\Local\Microsoft\WinGet\Links\scrcpy.exe").Target[0]
$directory = Split-Path -Path $filePath -Parent
$fileName = Split-Path -Path $filePath -Leaf

$adb = $directory+"\adb.exe"

$adbProcess = Get-Process adb.exe -ErrorAction SilentlyContinue
if (-not $adbProcess) 
{
    Start-Process -FilePath $adb -ArgumentList "start-server" -NoNewWindow -Wait
}
$result = Start-Process -FilePath $adb -ArgumentList "$Command $Params" -NoNewWindow -Wait
Write-Host $result`
you need rewrite $filePath .
i hope it is helpful for you

**Comment by hrz6976** [comment]
For me, the fix is to add the adb installation directory to PATH:
```
C:\Users<USERNAME>\AppData\Local\Microsoft\WinGet\Packages\Google.PlatformTools_Microsoft.Winget.Source_8wekyb3d8bbwe\platform-tools
```

**Comment by weirdal3333** [comment]
The platform tools folder MUST be in the path. Why isn't it being added to the path? That's a major thing in the installation instructions that Google has provided for over a decade. 
Who made this winget package?!

**Comment by aisgbnok** [comment]
- Fixed by #190804

**Comment by aisgbnok** [comment]
This has been resolved for all versions of the [`Google.PlatformTools`](https://github.com/microsoft/winget-pkgs/tree/master/manifests/g/Google/PlatformTools) package.
